kuma/core/whitenoise/KumaDjangoWhiteNoise.py

```diff
@@ -0,0 +1,12 @@
+from whitenoise.django import DjangoWhiteNoise
+
+
+class KumaDjangoWhiteNoise(DjangowhiteNoise):
+
+    def __init__(self, app):
+        DjangoWhiteNoise.__init__(self, app)
```

Comment:
What about `super(DjangoWhiteNoise, self).__init__(app)`? But also if you're not adding anything here there's no need, Python will do this for you.